sample-library-app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Comcast Cable Communications Management, LLC
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.xfinity.blueprint_sample_library_app">
+
+    <application
+        android:name=".BlueprintSampleApplication"
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name="com.xfinity.blueprint_sample_library_app.LauncherActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+        <activity android:name="com.xfinity.blueprint_sample_library_app.StaticScreenActivity" android:label="@string/static_screen"/>
+        <activity android:name="com.xfinity.blueprint_sample_library_app.DynamicScreenActivity" android:label="@string/dynamic_screen"/>
+        <activity android:name="com.xfinity.blueprint_sample_library.ArchitectSampleActivity" android:label="@string/bp_arch_screen"/>
```

Comment:
this should be com.xfinity.blueprint_sample_library_app.ArchitectSampleActivity

Reply:
Fixed